e2e/testcases/reconciler_manager_test.go

```diff
@@ -568,14 +636,43 @@ func TestAutopilotReconcilerAdjustment(t *testing.T) {
 		expectedTotalMemory = initialTotalMemory
 	}
 
-	nomostest.Wait(nt.T, "Waiting for third reconciliation", nt.DefaultWaitTimeout, func() error {
+	nomostest.Wait(nt.T, "resource requests to be updated", nt.DefaultWaitTimeout, func() error {
 		return nt.Validate(nomostest.DefaultRootReconcilerName, v1.NSConfigManagementSystem, &appsv1.Deployment{},
 			hasGeneration(generation), totalContainerCPURequestIs(expectedTotalCPU),
 			totalContainerMemoryRequestIs(expectedTotalMemory), firstContainerCPURequestIs(expectedFirstContainerCPU),
 			firstContainerMemoryRequestIs(expectedFirstContainerMemory))
 	})
 
 	// limit change
+	// Manually update resource limits
+	// The reconciler-manager does not manage the container resource limit if user does not override the resource limit through our API.
+	// In this case, we allow user manually update the memory limit.
```

Comment:
Can you update the comment to make it clear that we only allow manual update on the standard cluster. For Autopilot, the limit is the same as request.

Reply:
Yes, updated.